fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/email/EmailApiConstants.java

```diff
@@ -46,9 +46,10 @@
 
     public static final Set<String> UPDATE_REQUEST_DATA_PARAMETERS = new HashSet<>(Arrays.asList(messageParamName));
 
-    public static final String SMTP_SERVER = "SMTP_SERVER";
```

Comment:
Why do we have to rename this? Just curious.

Reply:
I made this to make the code more homogeneous and generic trying to show that the SMTP configuration is made there and is not only for Gmail (like it looks like it was.)